src/uct/ib/base/ib_iface.h

```diff
@@ -20,6 +20,8 @@
 #define UCT_IB_IFACE_NULL_RES_DOMAIN_KEY   0u
 #define UCT_IB_MAX_ATOMIC_SIZE             sizeof(uint64_t)
 
+#define UCT_IB_MAX_INLINE_VAL(_max_inline, _sub) \
```

Comment:
sounds like  ~
```
static inline uct_ud_iface_hdr_size(uct_ud_iface_t *iface, size_t min_size)
{
  <compare use iface->max_inline with min_size >
}
```

Reply:
replace macro by inline function?
BTW, it is used in RC too

Reply:
fixed